packages/pwa-kit-react-sdk/src/utils/ssr-server.test.js

```diff
@@ -819,29 +819,37 @@ describe('outgoingRequestHook tests', () => {
 
     const withHeaders = [true, false]
     const withCallback = withHeaders
+    const withProxyKeepAliveAgent = withCallback
 
     const testCases = []
     baseTestCases.forEach((baseTestCase) =>
         testMethods.forEach((testMethod) =>
             withHeaders.forEach((addHeaders) =>
                 withCallback.forEach((addCallback) => {
-                    const testCase = {...baseTestCase}
-                    testCase.name =
-                        `${testCase.name} via ${testMethod} ` +
-                        `${addHeaders ? 'with' : 'without'} headers, ` +
-                        `${addCallback ? 'with' : 'without'} callback`
-                    testCase.testMethod = testMethod
-                    testCase.addHeaders = addHeaders
-                    testCase.addCallback = addCallback
-                    testCases.push(testCase)
+                    withProxyKeepAliveAgent.forEach((addProxyKeepAliveAgent) => {
+                        const testCase = {...baseTestCase}
+                        testCase.name =
+                            `${testCase.name} via ${testMethod} ` +
+                            `${addHeaders ? 'with' : 'without'} headers, ` +
+                            `${addCallback ? 'with' : 'without'} callback`
```

Comment:
This is just a cosmetic suggestion to see more clearly what we are testing in the terminal output instead of seeing duplicated tests because the tests use the same name.

```
    ✓ non-loopback via url with headers, with callback (15 ms)
    ✓ non-loopback via url with headers, with callback (12 ms)
    ✓ non-loopback via url with headers, without callback (9 ms)
    ✓ non-loopback via url with headers, without callback (12 ms)
    ✓ non-loopback via url without headers, with callback (6 ms)
    ✓ non-loopback via url without headers, with callback (6 ms)
    ✓ non-loopback via url without headers, without callback (4 ms)
    ✓ non-loopback via url without headers, without callback (4 ms)
    ✓ non-loopback via host with headers, with callback (11 ms)
    ✓ non-loopback via host with headers, with callback (9 ms)
    ✓ non-loopback via host with headers, without callback (8 ms)
    ✓ non-loopback via host with headers, without callback (8 ms)
    ✓ non-loopback via host without headers, with callback (4 ms)
    ✓ non-loopback via host without headers, with callback (4 ms)
    ✓ non-loopback via host without headers, without callback (2 ms)
    ✓ non-loopback via host without headers, without callback (3 ms)
    ✓ non-loopback via hostname with headers, with callback (9 ms)
    ✓ non-loopback via hostname with headers, with callback (10 ms)
    ✓ non-loopback via hostname with headers, without callback (8 ms)
    ✓ non-loopback via hostname with headers, without callback (7 ms)
    ✓ non-loopback via hostname without headers, with callback (4 ms)
    ✓ non-loopback via hostname without headers, with callback (4 ms)
    ✓ non-loopback via hostname without headers, without callback (3 ms)
    ✓ non-loopback via hostname without headers, without callback (3 ms)
    ✓ loopback via url with headers, with callback (17 ms)
    ✓ loopback via url with headers, with callback (16 ms)
    ✓ loopback via url with headers, without callback (19 ms)
    ✓ loopback via url with headers, without callback (14 ms)
    ✓ loopback via url without headers, with callback (14 ms)
    ✓ loopback via url without headers, with callback (13 ms)
    ✓ loopback via url without headers, without callback (11 ms)
    ✓ loopback via url without headers, without callback (10 ms)
    ✓ loopback via host with headers, with callback (17 ms)
    ✓ loopback via host with headers, with callback (14 ms)
    ✓ loopback via host with headers, without callback (15 ms)
    ✓ loopback via host with headers, without callback (11 ms)
    ✓ loopback via host without headers, with callback (13 ms)
    ✓ loopback via host without headers, with callback (10 ms)
    ✓ loopback via host without headers, without callback (9 ms)
    ✓ loopback via host without headers, without callback (10 ms)
    ✓ loopback via hostname with headers, with callback (15 ms)
    ✓ loopback via hostname with headers, with callback (14 ms)
    ✓ loopback via hostname with headers, without callback (13 ms)
    ✓ loopback via hostname with headers, without callback (12 ms)
    ✓ loopback via hostname without headers, with callback (13 ms)
    ✓ loopback via hostname without headers, with callback (9 ms)
    ✓ loopback via hostname without headers, without callback (10 ms)
    ✓ loopback via hostname without headers, without callback (8 ms)
```  




```suggestion
                            `${addCallback ? 'with' : 'without'} callback, ` +
                            `${addProxyKeepAliveAgent ? 'with' : 'without'} KeepAliveAgent`
```